views/doc.cfm

```diff
@@ -288,6 +288,9 @@
 									<button class="example-btn btn btn-default" data-name="#encodeForHTMLAttribute(LCase(data.name))#" data-index="#example_index#"><span class="glyphicon glyphicon-play-circle"></span>&nbsp; Run Code</button>
 								</div>
 							</cfif>
+								<div class="pull-right" style="margin-right:10px">
+									<button class="copy-btn btn btn-default" onclick="copyTextToClipboard('#encodeForHTML(ex.code)#')" ><span class="glyphicon glyphicon-duplicate"></span>&nbsp; Copy Code</button>
```

Comment:
I feel like "glyphicon-copy" would be better here.

Reply:
updated